releasenotes/notes/dagcircuit-fix-substitute-node-global-phase-usage-ee05476a7d0b24c6.yaml

```diff
@@ -0,0 +1,12 @@
+---
+upgrade:
+  - |
+    The ``DAGCircuit.__eq__`` method, which is used to check structural equality
+    of :class:`~qiskit.dagcircuit.DAGCircuit` and :class:`~qiskit.circuit.QuantumCircuit` instances, has been updated to
+    include the ``global_phase`` and ``calibrations`` attributes in fields
+    checked for equality.
```

Comment:
See comment above, but this should be removed with the extra checking in `__eq__`

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/5673/commits/c71e2b1066eaaed53554122a10a2e6f2ea55283f